roles/gateway_common/tasks/initial-update-root-password.yml

```diff
@@ -2,7 +2,7 @@
 - name: Update expired root user password
   no_log: true # don't log decrypted passwords in responses
   expect:
-    command: sshpass -p '{{ gateway_ssgconfig_password }}' ssh {{ ansible_ssh_extra_args }} {{ ansible_user }}@{{ dest }}
+    command: sshpass -p '{{ gateway_ssgconfig_password }}' ssh {{ ansible_ssh_extra_args }} {{ ansible_user }}@{{ ansible_host }}
```

Comment:
dest is target gateway. It is now replaced with ansible_host?

Reply:
yes, the role shouldn't care about it is src or dest. it only cares about it has a machine to talk to. becuz src dest logical is very related to our inventory design..roles shouldn't know that part of the logic